wikipedia-dark.user.css

```diff
@@ -6546,6 +6547,49 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   .list-switcher-element a[role="button"] {
     color: var(--base-color) !important;
   }
+  tr > *[style*="background:#c0cfe4"],
+  tr > *[style*="background:#9999DF"] {
+    background-color: var(--blue-4) !important;
+  }
+  tr > *[style*="background:#a0ade3"],
+  tr > *[style*="background:#ccccff"] {
+    background-color: var(--blue-3) !important;
+  }
+  tr > *[style*="background:#e2e4c0"],
```

Comment:
`#e2e4c0` is on green specrum, Id'e probably go for `--green-2`

Reply:
Those are used in latin conjugation tables, if we move them to green I think there's not enough difference between the different sections anymore and I find this ugly... Compare

<details><summary>Current (colorful)</summary>

![Capture d’écran de 2021-03-11 09-15-48](https://user-images.githubusercontent.com/24524930/110756486-8ece2f00-824a-11eb-98e4-dfe4d781d40c.png)

</details>

<details><summary>With green-2 and green-1 instead of yellow/pink:</summary>


![Capture d’écran de 2021-03-11 09-16-25](https://user-images.githubusercontent.com/24524930/110756478-8c6bd500-824a-11eb-9200-98632e995a84.png)

</details>

Also I would name these color yellows, and apparently [this website agrees](https://www.colorhexa.com/e2e4c0): they name #e2e4c0 "light grayish yellow", and #e0e0b0 is "very soft yellow".